src/App.test.js

```diff
@@ -1,8 +1,38 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
-
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
-});
+// Imports React into our test file.
+import React from 'react'
+
+// Imports Enzyme testing and deconstructs Shallow into our test file.
+import Enzyme, { shallow } from 'enzyme'
+
+// Imports Adapter utilizing the latest react version into our test file so we can run a testing render on any component we may need.
+import Adapter from 'enzyme-adapter-react-16'
+
+// Imports in the component we are going to be testing.
+import App from './App'
+import NotFound from './pages/NotFound'
+
+//Allows us to utilize the adapter we import in earlier, allowing us to call and render a component.
+Enzyme.configure({adapter: new Adapter()})
+
+let renderedApp = shallow(<App />)
+    
+describe("When App renders", () => {
+  it("displays a header and footer", () => {
+    
+    const appHeading = renderedApp.find("Header")
+    expect(appHeading.length).toEqual(1)
+
+    const appFooter = renderedApp.find("Footer")
+    expect(appFooter.length).toEqual(1)
+  })
+
+  // testing for routes
```

Comment:
As your client, I would not like to see any code that is not being used. Please remove lines 29 - 38